examples/server_profile.go

```diff
@@ -20,14 +20,14 @@ func main() {
 		os.Getenv("ONEVIEW_OV_DOMAIN"),
 		os.Getenv("ONEVIEW_OV_ENDPOINT"),
 		false,
-		800,
+		1200,
 		"*")
 
 	initialScopeUris := new([]utils.Nstring)
 	*initialScopeUris = append(*initialScopeUris, utils.NewNstring("/rest/scopes/74877630-9a22-4061-9db4-d12b6c4cfee0"))
 
 	server_profile_create_map := ov.ServerProfile{
-		Type:              "ServerProfileV9",
+		Type:              "ServerProfileV11",
```

Comment:
are we not adding new fields like consistencyState in the struct?

Reply:
No, we are just minimal support for 1200.